src/pages/login/Login.jsx

```diff
@@ -1,37 +1,81 @@
-import React, { useState } from 'react';
-import { connect } from 'react-redux';
-import {
-  Typography,
-  Card,
-  Button,
-  TextField,
-  Stack,
-  Box,
-  FormControl,
-} from '@mui/material';
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Typography, Card, Button, TextField, Stack, Box, FormControl } from '@mui/material';
+import Snackbar from '@mui/material/Snackbar';
+import Alert from '@mui/material/Alert';
 import { fetchUserToken } from '../../store/commands';
-
 import './Login.sass';
 import { Footer } from '../../components';
 import { useMediaDown } from '../../components/utils';
 import { getFieldLabel } from '../../utils';
 
-const Login = (props) => {
+const Login = () => {
   const smallScreen = useMediaDown('sm');
+  const dispatch = useDispatch();
+  const authorization = useSelector((state) => state.authorization);
+  const [emailIsValid, setEmailIsValid] = useState(true);
+  const [passwordIsValid, setPasswordIsValid] = useState(true);
+  const [openToaster, setOpenToaster] = React.useState(false);
+  const [errorMessages, setErrorMessages] = React.useState([]);
```

Comment:
Create a custom hook for using toaster that returns isToasterOpen(boolean), addToast(function, accepts a message and adds a message to the array), handleCloseToaster. Put it into the same folder as alerts component and please make it reusable (for any clarifications text me)